docs/documentation/advanced/term/term.mdx

```diff
@@ -4,7 +4,11 @@ title: Term
 
 ## Basic Usage
 
-Matches documents containing a specified [term](/documentation/concepts/term).
+Matches documents containing a specified [term](/documentation/concepts/term). Term queries are used when you need to match values exactly as they are stored in the index, without any analysis or tokenization.
```

Comment:
I like the idea of adding a "this should be used when Y" phrase to our "Basic Usage" sections.

Nitpick, but I think this would read better:

"Term queries should be used when looking to match values exactly as they are stored in the index, without any analysis or tokenization."

Reply:
Sure your suggestions seems better in terms of reading. I will update this.